content/blog/2020-08-28-core-devs-call-2020-Q3-Hardfork-Evaluation-Consensus-Finding/index.en.md

```diff
@@ -27,6 +33,6 @@ Topic discussion references:
 
 ### Recording of Core Devs Call
 
-* Link to be added after call.
+* [https://youtu.be/4W1l5krLPqI](https://youtu.be/4W1l5krLPqI)
```

Comment:
the link does't work. Here is it: https://www.youtube.com/watch?v=4W1l5krLPqI&t=5s

Reply:
Updated link to https://www.youtube.com/watch?v=4W1l5krLPqI